### PR TITLE
Always bottom footer #39

### LINKED ISSUE #39: Always bottom footer
image

https://www.figma.com/file/OZhSt60CA5rinmbr5aTfSP/Taskany?node-id=47%3A457

**NB:** year must be autogenerated.

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
Resolves #39

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+const StyledFooter = styled.footer`
+    padding: 20px 20px;
+    display: flex;
+    justify-content: start;
+    max-height: 60px;
+`;
+
+const StyledFooterText = styled.span`
+    font-weight: 400;
```
Comment (awinogradov) [inline src/components/Footer.tsx:12]: This is default weight. Unnecessary here.

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    color: rgba(255, 255, 255, 0.6);
+`;
+
+export const Footer: React.FC = () => {
+    return (
+        <>
```
Comment (awinogradov) [inline src/components/Footer.tsx:18]: Unnecessary wrap.

File: src/components/Page.tsx [hunk 3]
```diff
@@ -5,22 +5,27 @@ import { pageContext } from '../utils/pageContext';
 import { ExternalPageProps } from '../utils/ssrProps';
 
 import { Header } from './Header';
+import { Footer } from './Footer';
+import styled from 'styled-components';
```
Comment (awinogradov) [inline src/components/Page.tsx:9]: Looks like incorrect imports order. Do you some errors from ESLint in your editor?

File: src/components/Page.tsx [hunk 4]
```diff
@@ -5,22 +5,27 @@ import { pageContext } from '../utils/pageContext';
...
 
 interface PageProps {
     locale: ExternalPageProps['locale'];
     title: string;
 }
 
+const StyledContent = styled.div`
+    min-height: calc(100vh - 160px);
```
Comment (awinogradov) [inline src/components/Page.tsx:17]: Would be better to add comment about 160px. The next collaborator will not understand what is it.

File: src/components/SheepLogo.tsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+
+export const SheepLogo: React.FC = () => {
+
+    return (
```
Comment (awinogradov) [inline src/components/SheepLogo.tsx:6]: Does this is result SVG after SVGO optimisations?

**Review by awinogradov — CHANGES_REQUESTED** [review]
@Katrin-kudryash thanks for your contribution! ❤️ 

Some mistakes in your PR:
- you have duplicated commits. You must have 2 instead of 4.
- PR's title must be same with issue
- PR's description must have connection to issue. Ex: `Resolves #39`.

File: src/components/SheepLogo.tsx [hunk 6]
```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+
+export const SheepLogo: React.FC = () => {
+
```
Comment (awinogradov) [inline src/components/SheepLogo.tsx:5]: Unnecessary empty string

Comment (Katrin-kudryash) [reply src/components/SheepLogo.tsx]: done

File: src/components/Page.tsx (hunk 4 above)
Comment (Katrin-kudryash) [reply src/components/Page.tsx:18]: done

File: src/components/Page.tsx (hunk 3 above)
Comment (Katrin-kudryash) [reply src/components/Page.tsx]: done

File: src/components/Footer.tsx (hunk 2 above)
Comment (Katrin-kudryash) [reply src/components/Footer.tsx:18]: done

File: src/components/Footer.tsx (hunk 1 above)
Comment (Katrin-kudryash) [reply src/components/Footer.tsx]: done

File: src/components/SheepLogo.tsx (hunk 5 above)
Comment (Katrin-kudryash) [reply src/components/SheepLogo.tsx:4]: done

File: src/components/Footer.tsx (hunk 2 above)
Comment (awinogradov) [reply src/components/Footer.tsx:18]: Nope. You have root element `<StyledFooter>` and you wrap it with `<></>` — this is unnecessary.

**Review by awinogradov — CHANGES_REQUESTED** [review]

**Comment by Katrin-kudryash** [comment]
Resolves #39

**Review by awinogradov — APPROVED** [review]
Awesome 🤩

**Pull request merged**